CI dependency graph build step was failing and is now disabled

While working on #2003, I had to disable the module dependency graph creation on CI as it was consistently failing. 

Investigate why these failures occurred and re-enable the graph creation step.

It failed a few times with: 

```
##[debug]Evaluating condition for step: 'Check Graphs'
##[debug]Evaluating: success()
##[debug]Evaluating success:
##[debug]=> true
##[debug]Result: true
##[debug]Starting: Check Graphs
##[debug]Loading inputs
##[debug]Loading env
Run git add -- '**/README.md' && git diff --cached --quiet --exit-code -- '**/README.md'
##[debug]/usr/bin/bash -e /home/runner/work/_temp/2689b210-b4b0-4222-92b6-40e583afee95.sh
Error: Process completed with exit code 1.
##[debug]Finishing: Check Graphs
```

Example from https://github.com/android/nowinandroid/pull/1902: https://github.com/android/nowinandroid/actions/runs/19750372536/job/56593641923

--- a/.github/workflows/Build.yaml
+++ b/.github/workflows/Build.yaml
@@ -79,6 +79,7 @@ jobs:
 
       - name: Update Graphs
         run: ./gradlew graphUpdate
+        continue-on-error: true
 
       - name: Check Graphs
         id: graphs_verify

